### ISSUE TITLE
Open link in the same tab

### ISSUE THREAD

**Issue by megamaster4** [Dalvany/dalvany-image-panel#62]
Hi,

Would it be possible to open a link in the same tab, instead of opening a new tab?

I saw that the following code decides to open the link in a new tab:
```javascript
<div className={'div-container'} style={{ width: w, overflow: 'hidden' }}>
      <ConditionalWrapper
        condition={link !== undefined}
        wrapper={(children) => (
          <a href={link.link} target={'_blank'} rel={'noreferrer noopener'} style={{ height: '100%' }}>
            {children}
          </a>
        )}
```
Changing `target={'_blank'}` to `target={'_self'}` would achieve my goal for opening the link in the same tab, but I can imagine that you don't want to change it in this way.

Is there a possibility to create an extra radio button to make this a changable feature?

Many thanks in advance!

**Comment by Dalvany** [comment]
Yes, I'll do that. I rewrite your issue to make the code more readable.

**Comment by Dalvany** [comment]
Could you test [this pre-release](https://github.com/Dalvany/dalvany-image-panel/releases/download/v2.6.0/dalvany-image-panel-2.6.0.zip) to see if it's ok for your use ?

**Comment by Dalvany** [comment]
You're welcome. I'll make a proper release on grafana soon I think.
